config/webhook/manifests.yaml

```diff
@@ -15,6 +15,8 @@ webhooks:
   failurePolicy: Ignore
   name: validate-externalip.webhook.svc
   sideEffects: None
+  admissionReviewVersions:
+  - v1beta1
```

Comment:
This webhook implementation depends on controller-runtime v0.5.0 (based on kubernetes v1.17) which only supports AdmissionReview v1beta1.